src/clr-core/modal/modal.element.scss

```diff
@@ -0,0 +1,103 @@
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/utils';
+@import './../styles/utils/close';
+
+:host {
+  --backdrop-opacity: 0.85;
+  --backdrop-color: #{$cds-token-color-neutral-900};
+  --box-shadow-color: #{$cds-token-color-action-500};
+  --border-radius: #{$cds-token-global-border-radius};
+  --border-color: #{$cds-token-color-action-600};
+  --background-color: #{$cds-token-color-neutral-0};
+  --close-icon-color: #{$cds-token-color-neutral-600};
+  --close-icon-color-hover: #{$cds-token-color-neutral-700};
+  --content-box-shadow-color: #{$cds-token-color-neutral-1000};
+  --width: calc(8 * #{$cds-token-space-size-13});
```

Comment:
Some nits on the css api to make it a bit more flexible and follow the naming convention

`--backdrop-color` => `--backdrop-background`
`--box-shadow-color` => `--box-shadow`
`--background-color` => `--background`
`--border-color` => `--border`
`--content-box-shadow-color` => `--content-box-shadow`

We should lean towards shorthand props `--background` that way consumers have full control and we dont have to expose additional background-* props.

Reply:
I'll make the changes.. `--content-box-shadow-color` is a bit different though as it literally only lets you customize the color:

```
:host {
  --content-box-shadow-color: rgba(0, 0, 0, 0.2);
}

.modal-content {
  box-shadow: 0 $cds-token-space-size-1 $cds-token-space-size-2 $cds-token-space-size-2 var(--content-box-shadow-color);
}
```